src/wasm-interpreter.h

```diff
@@ -2408,6 +2423,36 @@ template<typename GlobalManager, typename SubType> class ModuleInstanceBase {
     // Stack of <caught exception, caught catch's try label>
     SmallVector<std::pair<WasmException, Name>, 4> exceptionStack;
 
+  protected:
+    std::pair<SubType*, Name> getTableInstance(Name tableName) {
```

Comment:
These APIs returning a pair is unintuitive to me - perhaps add a comment?

Reply:
I agree. I removed the two that returned a pair for now, until I can come up with a better way.